include/sir/platform.h

```diff
@@ -450,13 +452,16 @@ _set_thread_local_invalid_parameter_handler(
 #   include <stdatomic.h>
 #  endif
 #  if defined(SIR_SYSLOG_ENABLED)
-#   include <syslog.h>
+#   if !defined(__cplusplus)
```

Comment:
Why if !defined C++?

Reply:
On some platforms including the syslog C header directly within C++ is unsupported.  PASE is one.  I should make that just for PASE?